core/src/test/java/cz/xtf/core/image/ImageTest.java

```diff
@@ -99,4 +99,50 @@ public void testNotResolvableImage() {
         Assertions.assertThrows(UnknownImageException.class, () -> Image.resolve("blabla"),
                 "Expected UnknownImageException to be thrown for unknown image.");
     }
+
+    @Test
+    public void testImageFromUrlOneToken() {
+        Image image = Image.from("nginx");
+        Assertions.assertTrue(image.getRegistry().isEmpty(), "Registry for image url: 'nginx' must be empty.");
+        Assertions.assertTrue(image.getUser().isEmpty(), "User for image url: 'nginx' must be empty.");
+        Assertions.assertEquals("nginx", image.getRepo(), "Wrong repository name.");
+        Assertions.assertTrue(image.getTag().isEmpty(), "Tag for image url: 'nginx' must be empty.");
+    }
+
+    @Test
+    public void testImageFromUrlTwoTokens() {
+        Image image = Image.from("user/nginx");
+        Assertions.assertTrue(image.getRegistry().isEmpty(), "Registry for image url: 'user/nginx' must be empty.");
+        Assertions.assertEquals(image.getUser(), "user", "User for image url: 'user/nginx' is wrong.");
+        Assertions.assertEquals("nginx", image.getRepo(), "Wrong repository name.");
```

Comment:
weird order of expected/actual

> [assertEquals](https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertEquals-byte-byte-java.lang.String-)(byte expected, byte actual, [String](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html?is-external=true) message)

below occurrences as well

Reply:
Thanks, fixed.